app/templates/gentelella/users/settings/pages/applications.html

```diff
@@ -14,13 +14,12 @@
 {% block content %}
 
     <div style="display: block;">
-        {% if user.user_detail.facebook %}
         <div class="row" style="margin-bottom:20px">
 
             <div class="col-md-6 col-xs-6">
                 <label>Facebook:</label>
                 <div class="input-group">
-                    <span class="input-group-addon">facebook.com/</span>
+                    <span class="input-group-addon minwidth">facebook.com/</span>
```

Comment:
Don't give class names like this. Give a class name that is understandable and reusable. Like maybe `social-addon` or something like that.